mmpose/apis/inference_3d.py

```diff
@@ -6,7 +6,71 @@
 from .inference import LoadImage, _box2cs, _xywh2xyxy, _xyxy2xywh
 
 
-def _collate_pose_sequence(pose_results, with_track_id=True):
+def _gather_pose_lifter_inputs(pose_results,
+                               bbox_center,
+                               bbox_scale,
+                               norm_pose_2d=False):
+    """Gather input data (keypoints and track_id) for pose lifter model.
+
+    Notes:
+        T: The temporal length of the pose detection results
+        N: The number of the person instances
+        K: The number of the keypoints
+        C: The channel number of each keypoint
+
+    Args:
+        pose_results (List[List[Dict]]): Multi-frame pose detection results
+            stored in a nested list. Each element of the outer list is the
+            pose detection results of a single frame, and each element of the
+            inner list is the pose information of one person, which contains:
+                keypoints (ndarray[K, 2 or 3]): x, y, [score]
+                track_id (int): unique id of each person, required when
+                    ``with_track_id==True```
+                bbox ((4, ) or (5, )): left, right, top, bottom, [score]
```

Comment:
We still have `bbox` in `pose_results`, right?

Reply:
This function only keeps keys that are necessary to 2D-3D lifting. So `bbox` is removed from `pose_results` in this function. For visualization, we can get `bbox` from 2D pose detection results.

Reply:
OK. I see.